lib/node_modules/@stdlib/string/base/replace-before-last/docs/types/test.ts

```diff
@@ -0,0 +1,69 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import replaceBeforeLast = require( './index' );
+
+
+// TESTS //
+
+// The function returns a string...
+{
+	replaceBeforeLast( 'beep boop', ' ', 'foo', 'beep boop'.length ); // $ExpectType string
+	replaceBeforeLast( 'beep boop', 'xyz', 'foo', 'beep boop'.length ); // $ExpectType string
+	replaceBeforeLast( 'beep boop', '', 'foo', 'beep boop'.length ); // $ExpectType string
```

Comment:
```suggestion
	replaceBeforeLast( 'beep boop', ' ', 'foo', 10 ); // $ExpectType string
	replaceBeforeLast( 'beep boop', 'xyz', 'foo', 10 ); // $ExpectType string
	replaceBeforeLast( 'beep boop', '', 'foo', 10 ); // $ExpectType string
```

We only checking types here, so what the value actually is doesn't matter.